vue/components/organisms/pickers/pickers.vue

```diff
@@ -13,7 +13,7 @@
                 class="button-scroll button-scroll-left button-scroll-parts button-scroll-parts-left"
                 v-if="enableScrollParts"
                 v-show="scrollParts"
-                v-on:click="slideLeftParts"
+                v-on:click.stop="slideLeftParts"

```

Comment:
Why is this needed?

Reply:
So that this behaviour (gif) doesn't happen. I think it is a bit weird that by clicking an arrow to scroll horizontally, the panel opens. It should only open when clicking a part.
![pickers-stop](https://user-images.githubusercontent.com/25725586/99959982-e6bc5500-2d83-11eb-9997-6052e0edc5c5.gif)